components/homepage/HowItWorksBlock.tsx

```diff
@@ -148,7 +148,7 @@ const HowItWorksBlock = (props: any) => {
                             [`@media screen and (min-width: ${theme.breakpoints[4]})`]:
                                 {
                                     width: "57.6rem !important",
-                                    py: "7.4rem !important",
+                                    // py: "7.4rem !important",
```

Comment:
Should we keep this comment?

Reply:
Addressed in 00ab6e73c6e7d0a5fc453fc39db06032ab7b4b39

Reply:
Addressed in 00ab6e73c6e7d0a5fc453fc39db06032ab7b4b39